website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,387 @@
+Usually the problematic instance can be identified as the respective `Machine` CRD has status `pending`.
+
+The instance / node name can be obtained from the `Machine` `.status` field:
+
+```
+$ kubectl get machine <machine-name> -o json | jq -r .status.node
+```
+
+This is all the information needed to go ahead and use `gardenctl ssh` to get a shell to the node. 
+In addition, the used cloud provider, the specific identifier of the instance and the instance region can be identified from the `Machine` CRD. 
+
+Get the identifier of the instance via:
+```
+$ kubectl get machine <machine-name> -o json | jq -r .spec.providerID // e.g aws:///eu-north-1/i-069733c435bdb4640
+```
+The identifier shows that the instance belongs to the cloud provider `aws` with the ec2 instance-id `i-069733c435bdb4640` in region `eu-north-1`.
+
+To get more information about the instance, check out the `MachineClass` (e.g `AWSMachineClass`) that is associated with each `Machine` CRD in the `Shoot` namespace of the `Seed` cluster.
+The `AWSMachineClass` contains the machine image ([ami](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)), machine-type, iam information, network-interfaces, subnets, security groups and attached volumes.
+
+Of course, the information can also be used to get the instance with the cloud provider CLI / API.
+
+## gardenctl ssh
+
+Using the node name of the problematic instance, we can use the `gardenctl ssh` command to get SSH access to the cloud provider 
+instance via an automatically set up [bastion host](https://en.wikipedia.org/wiki/Bastion_host).
+`gardenctl` takes care of spinning up the `bastion` instance, setting up the SSH keys, ports and security groups and opens a root shell on the target instance.
+After the SSH session ended, `gardenctl` deletes the created cloud provider resources.
+
+Use the following commands:
+
+First, target a Garden cluster containing all the Shoot definitions.
+```
+$ gardenctl target garden <target-garden>
+```
+
+Target an available Shoot by name. 
+This sets up the context, configures the `kubeconfig` file of the Shoot cluster and downloads the cloud provider credentials.
+Subsequent commands will execute in this context.
+
+```
+$ gardenctl target shoot <target-shoot>
+```
+
+This uses the cloud provider credentials to spin up the bastion and to open a shell on the target instance.
+```
+$ gardenctl ssh <target-node>
+```
+
+## SSH using a manually created Bastion on AWS
+
+In case you are not using `gardenctl` or want to control the bastion instance yourself, you can also manually set it up. 
+The steps described here are generally [the same as `gardenctl` uses internally](https://github.com/gardener/gardenctl/blob/10a537942b94234914758c0f6d053dc1cf218ecd/pkg/cmd/ssh_aws.go#L53-L52).
+The details of setting up the bastion host is different from cloud provider to cloud provider. 
+It can be generalized into the following steps:
```

Comment:
There are a few issues with redundancy and cohesion here. How about simplifying to this:

```suggestion
The steps described here are generally the same as [those used by `gardenctl` internally](https://github.com/gardener/gardenctl/blob/10a537942b94234914758c0f6d053dc1cf218ecd/pkg/cmd/ssh_aws.go#L53-L52).
Despite some cloud provider specifics they can be generalized to the following list: 
```